pumpio/pumpio.php

```diff
@@ -326,10 +328,10 @@ function pumpio_settings(&$a, &$s)
 	$s .= '<div class="settings-submit-wrapper" ><input type="submit" id="pumpio-submit" name="pumpio-submit" class="settings-submit" value="' . L10n::t('Save Settings') . '" /></div></div>';
 }
 
-function pumpio_settings_post(&$a, &$b)
+function pumpio_settings_post(App $a, array &$b)
 {
-	if (x($_POST, 'pumpio-submit')) {
-		if (x($_POST, 'pumpio_delete')) {
+	if (x($_POST,'pumpio-submit')) {
+		if (x($_POST,'pumpio_delete')) {
```

Comment:
Why did you remove the spaces in the two previous lines? Please keep them as is.

Reply:
Ops? My fault. Okay, I revert them. And I can switch here away from `x()` (deprecated) to `!empty()` okay?

Reply:
By all means!